tests/LINEBot/SenderTest.php

```diff
@@ -0,0 +1,350 @@
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+namespace LINE\Tests\LINEBot;
+
+use LINE\LINEBot\ImagemapActionBuilder\AreaBuilder;
+use LINE\LINEBot\ImagemapActionBuilder\ImagemapMessageActionBuilder;
+use LINE\LINEBot\ImagemapActionBuilder\ImagemapUriActionBuilder;
+use LINE\LINEBot\MessageBuilder\AudioMessageBuilder;
+use LINE\LINEBot\MessageBuilder\Imagemap\BaseSizeBuilder;
+use LINE\LINEBot\MessageBuilder\ImagemapMessageBuilder;
+use LINE\LINEBot\MessageBuilder\ImageMessageBuilder;
+use LINE\LINEBot\MessageBuilder\LocationMessageBuilder;
+use LINE\LINEBot\MessageBuilder\StickerMessageBuilder;
+use LINE\LINEBot\MessageBuilder\TemplateBuilder\ButtonTemplateBuilder;
+use LINE\LINEBot\MessageBuilder\TemplateMessageBuilder;
+use LINE\LINEBot\MessageBuilder\TextMessageBuilder;
+use LINE\LINEBot\MessageBuilder\VideoMessageBuilder;
+use LINE\LINEBot\QuickReplyBuilder\ButtonBuilder\QuickReplyButtonBuilder;
+use LINE\LINEBot\QuickReplyBuilder\QuickReplyMessageBuilder;
+use LINE\LINEBot\SenderBuilder\SenderMessageBuilder;
+use LINE\LINEBot\TemplateActionBuilder\CameraRollTemplateActionBuilder;
+use LINE\LINEBot\TemplateActionBuilder\CameraTemplateActionBuilder;
+use LINE\LINEBot\TemplateActionBuilder\PostbackTemplateActionBuilder;
+use LINE\LINEBot\TemplateActionBuilder\UriTemplateActionBuilder;
+use PHPUnit\Framework\TestCase;
+
+class SenderTest extends TestCase
+{
+    public function testTextMessageWithSender()
+    {
+        $sender = new SenderMessageBuilder("test1", "https://example.com/test2");
+
+        $messageTemplate = new TextMessageBuilder('test text1', 'test text2', $sender);
+
+        $this->assertEquals(
+            [
+                [
+                    'type' => 'text',
+                    'text' => 'test text1',
+                    'sender' => [
+                        'name' => 'test1',
+                        'iconUrl' => 'https://example.com/test2',
+                    ],
+                ],
+                [
+                    'type' => 'text',
+                    'text' => 'test text2',
+                    'sender' => [
+                        'name' => 'test1',
+                        'iconUrl' => 'https://example.com/test2',
+                    ],
+                ],
+            ],
+            $messageTemplate->buildMessage()
+        );
+    }
+
+    public function testStickerMessageWithSender()
+    {
+        $sender = new SenderMessageBuilder(null, "https://example.com/test2");
+
+        $messageTemplate = new StickerMessageBuilder('1', '1', null, $sender);
+
+        $this->assertEquals(
+            [
+            [
```

Comment:
Fix indent please. And also other tests below.

Reply:
I formatted these styles